<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Connect</class>
 <widget class="QWidget" name="Form_Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Widget_Config" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_addr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title_port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_addr">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_port">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="BACKGROUND">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="Widget_Connect" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>重连</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_config">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Font: 40pt;</string>
    </property>
    <property name="text">
     <string>连接似乎出错了</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>BACKGROUND</zorder>
  <zorder>Widget_Config</zorder>
  <zorder>Widget_Connect</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
